client/datatypes.go

```diff
@@ -14,6 +14,12 @@ package client
 
 import "net/http"
 
+// Field user type.
+type Field struct {
```

Comment:
@kwk You can make an issue in this repo, something that match a 'task to be done' related to some user story and add the link for future reference.